distribution/src/config/jvm.options

```diff
@@ -42,9 +42,7 @@
 # following three lines to your version of the JDK
 # 8-13:-XX:-UseConcMarkSweepGC
 # 8-13:-XX:-UseCMSInitiatingOccupancyOnly
-14-:-XX:+UseG1GC
-14-:-XX:G1ReservePercent=25
-14-:-XX:InitiatingHeapOccupancyPercent=30
+# 14-:-XX:+UseG1GC
```

Comment:
Since we are not choosing the actual GC implementation in ergonomics, doesn't this need to stay uncommented?

Reply:
Or if we are relying on G1GC being the default now, then do we still need this line in the jvm options at all? Will the ergonomics detect using G1GC when it is not explicitly added to the options?

Reply:
Thank you for catching this. I should've uncommented the line `14-:-XX:+UseG1GC`. I think we still want to keep it uncommented in case someone wants to use G1GC with jdk older than jdk14. Then the comment above still makes sense.